tests/src/integration/tests/test_auth.cpp

```diff
@@ -118,7 +118,7 @@ CASSANDRA_INTEGRATION_TEST_F(AuthenticationTests, InvalidEmptyCredentials) {
      * a simple form. This test serves to characterize what is there presently.
      */
     Session session = connect_using_credentials(i, "", "");
-    ASSERT_EQ(session.connect_error_code(), CASS_ERROR_LIB_NO_HOSTS_AVAILABLE);
+    ASSERT_EQ(session.connect_error_code(), CASS_ERROR_SERVER_BAD_CREDENTIALS);
```

Comment:
Test was reporting this error rather than "no hosts available" and it seems more descriptive anyway